starter-code/index.js

```diff
@@ -1,11 +1,47 @@
 class SortedList {
-  constructor() {}
-  add(item) {}
-  get(pos) {}
-  max() {}
-  min() {}
-  average() {}
-  sum() {}
+  constructor() {
+    this.items  = [];
+    this.length = 0;
+  }
+  add(x){
+    this.items.push(x);
+    this.length++;
+  }
+
+  get(x){
```

Comment:
for this, you're close. The test wanted
 ```
  get(x) {
    if (this.items.length === 0) {
      throw new Error("OutOfBounds");
    }
    return this.items[x - 1];
  }
```

for the position, you need to -1 because the index of an array starts at 0. Without doing -1, you get an off by one error